backend/db/ddl-17-concept_ancestor_plus.jinja.sql

```diff
@@ -0,0 +1,23 @@
+-- Table: concept_ancestor_plus ------------------------------------------------------------------------------------
```

Comment:
@Sigfried Put new `concept_ancestor_plus` in its own DDL. Added some missing things in there like `{{schema}}`, `{{optional_suffix}}`, and `{{optional_index_suffix}}`.